tests/docs/test_docs.py

```diff
@@ -7,15 +7,15 @@
 class Doc_Test(unittest.TestCase):
     @property
     def path_to_docs(self):
-        dirname, filename = os.path.split(os.path.abspath(__file__))
+        dirname, file_name = os.path.split(os.path.abspath(__file__))
         return dirname.split(os.path.sep)[:-2] + ["docs"]
 
     def test_html(self):
         wd = os.getcwd()
         os.chdir(os.path.sep.join(self.path_to_docs))
 
         if platform.system() != "Windows":
-            response = subprocess.run(["make", "html-noplot"])
+            response = subprocess.run(["make", "html"])
```

Comment:
This line, and the one below it, is why the doc test is being timed out on azure, please change it back. All of the examples and tutorials are tested separately from the doc build.